src/index.js

```diff
@@ -1,13 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
-import store from './store/index';
 import App from './components/App.jsx';
 import './styles/index.scss';
 
 ReactDOM.render(
-	<Provider store={store}>
-		<App />
-	</Provider>,
+		<App />,
```

Comment:
Why did you take out the store connection?

Reply:
I was not using the redux store, actions or reducers. So, when it is merged others have everything it will have the store back.

Reply:
@jo-rdan you don't need to delete them even if you aren't using it. it doesn't affect your work, just leave the files there, just the way they are.

Reply:
Don't forget someone work on setting up redux, so you removing it simply means, you are destroying what the person work on.

Reply:
Ok I will put them back